zinit-install.zsh

```diff
@@ -1683,6 +1683,13 @@ ziextract() {
             }
             ;;
         ((#i)*.bz2|(#i)*.bzip2)
+            # Rename file if its extension does not match "bz2". bunzip2 refuses
+            # to operate on files that are not named correctly.
+            # See https://github.com/zdharma-continuum/zinit/issues/105
+            if [[ $file != (#i)*.bz2 ]] {
+                command mv $file $file.bz2
+                file=$file.bz2
+            }
             →zinit-extract() { →zinit-check bunzip2 "$file" || return 1
```

Comment:
Looks like a missed Unicode char

Reply:
"missed" by whom?
This is a function name!

Reply:
https://github.com/zdharma-continuum/zinit/issues/58 only covered file names

Reply:
Ah, gotcha. We should spin off a task for removing those from function names. I don't think we should have any unicode chars in the codebase except for strings we're printing for the user. It makes the code a lot harder to work with

Reply:
https://github.com/zdharma-continuum/zinit/issues/108